### DISCUSSION TITLE
Gestionar dispositivos

### DISCUSSION THREAD

**Discussion by Elpanteradany** [Paco8/plugin.video.movistarplus#73] [General]
Buenas noches
Antes que nada darte la enhorabuena y las gracias por tus addons, me han descubierto un mundo dentro de kodi. 
Mi pregunta es: Porque en la opcion gestionar dispositivos de orange puedo elegir otro dispositivo donde este abierta la sesion(como por ejemplo el decodificador o un firestick) ya que asi por ejemplo puedo ver dazn en dos ubicaciones distintas y en la de movistar no?
Lo mismo es una pregunta muy estupida pero me resulta raro. Gracias de antemano

**Comment by Paco8** [comment]
En el addon de movistar me parece que también se puede cambiar de dispositivo.

**Reply by Elpanteradany** [reply]
Lo intento pero siempre se me queda señalado el dispositivo Pc (que en este caso es un android tv)No me deja señalar otro

**Comment by Paco8** [comment]
Lo que ocurre es que si seleccionas un dispositivo que no sea "PC/Mac" ese dispositivo se cambia a "PC/Mac". No es posible hacerse pasar por otro tipo de dispositivo diferente.

Si el problema que tienes es que te sale el mensaje de que se han alcanzado el número máximo de reproducciones, lo que puedes hacer es cerrar la sesión a un dispositivo que esté reproduciendo algo (la reproducción no se corta en ese dispositivo, creo), y así ya te dejará ver algo.

**Reply by Elpanteradany** [reply]
Lo probaré, Gracias
